FHEM/00_SIGNALduino.pm

```diff
@@ -2906,7 +2914,11 @@ sub SIGNALduino_Attr(@) {
 			Log3 $name, 3, "$name: Attr, Disable eventlogging";
 		}
 	}
-
+	elsif ($aName eq "userReadings")	# look reserved cc1101 readings
+	{
+		return "Note, please use other userReadings names.\nReserved names from $name are: cc1101_config, cc1101_config_ext, cc1101_patable"
+			if ($aVal =~ /cc1101_(config(_ext)?|patable)(\s|{)/);
```

Comment:
Die capture group brauchen nicht, daher non capturing group verwenden:
`if ($aVal =~ /cc1101_(?:config(?:_ext)?|patable)(?:\s|{)/);`

Reply:
Wenn wir schon so eine Warnung einbauen, müssten eigentlich alle selbst erzeugten Readings überwacht werden, wie z.B.: cmds, config, freeram, ping, state, uptime u.s.w.

Reply:
@sidey79 es ist glaube besser wenn wir diese Warnung herauslassen und nur die 3 Readings löschen.

In keinem Modul ist eine Prüfung drin ob der User bei den Attributen "userReadings" ein vorhandenes Reading überschreibt. Man geht davon aus, der User weiß was er dort bewußt macht.

Reply:
Wissen die es denn wirklich?

Reply:
Scheinbar möchtest du der erste sein, der in seinem Modul dem User den Hinweis gibt sollte er nicht dran denken :-) Du kannst gern sämtliche Module nachsehen aber ich habe nichts dazu gefunden. Ebenso in keiner Wiki oder Quelle erlese ich den Hinweis, auf ein `"Achtung, es könnten Modulinterne Readings"` überschrieben werden.

Gern lasse ich mich des besseren belehren wenn irgendwo eine solche Prüfung existiert.

Reply:
Ich habe keine Ahnung ob irgendwer daran gedacht hat. Dann machen wir es halt einfach sow ie alle.

Reply:
Es ist der beste Weg, so wie alle zu machen. Alles andere, da wären wir Vorreiter und vermutlich die Einzigen.